nncf/quantization/algorithms/weight_compression/weight_lowering.py

```diff
@@ -154,7 +154,7 @@ def calculate_signed_scale(weight: Tensor, reduction_axes: ReductionAxes, num_bi
     w_max = fns.max(weight, axis=reduction_axes, keepdims=True)
 
     scale = fns.where(w_abs_min >= w_max, w_abs_min, -w_max)
-    scale /= level_high
+    scale *= 1.0 / level_high
```

Comment:
I have changed every division operation in the compression pipeline because this is how it will work inside an openvino model during scale computation. This way we'll have a complete alignment. 

As I understand every division operation inside an OV graph is performed in the form of `a * (1.0 / b)`. This also applies the the operation you mention.